### PR TITLE
[zh-cn]: init translation for `NotificationEvent.action`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

part of the #18099

### CODE REVIEW

File: files/zh-cn/web/api/notificationevent/action/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: NotificationEvent：action 属性
+slug: Web/API/NotificationEvent/action
+l10n:
+  sourceCommit: 212e53bbdaf88af59a57ec1f335bde5e828b68c5
+---
+
+{{APIRef("Web Notifications")}}
+
+{{domxref("NotificationEvent")}} 接口的 **`action`** 只读属性返回用户单击的通知按钮的字符串 ID。如果用户单击操作按钮以外的通知，或者通知没有按钮，则此值返回空字符串。通知 ID 在创建通知期间通过 actions 数组属性设置，并且无法修改，除非通知被替换。
```
Comment (yin1999) [inline files/zh-cn/web/api/notificationevent/action/index.md:10]:
```suggestion
{{domxref("NotificationEvent")}} 接口的 **`action`** 只读属性返回用户单击的通知按钮的字符串 ID。如果用户单击通知除操作按钮以外的区域，或者通知没有按钮，则此值返回空字符串。通知 ID 在创建通知期间通过 actions 数组属性设置，并且无法修改（除非通知被替换）。
```

**Review by yin1999 — COMMENTED** [review]
Please check the suggestion (I think the clicked area is still on the notification area)

**Comment by skyclouds2001** [comment]
> Please check the suggestion (I think the clicked area is still on the notification area)

Yes, truly it is. The suggestion is better.

**Review by yin1999 — APPROVED** [review]
Thanks 🙏 @skyclouds2001

**Pull request merged**